ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/worldview/BonsaiWorldStateUpdateAccumulator.java

```diff
@@ -65,35 +70,36 @@ public class BonsaiWorldStateUpdater extends AbstractWorldUpdater<BonsaiWorldVie
   private final Map<Address, StorageConsumingMap<BonsaiValue<UInt256>>> storageToUpdate =
       new ConcurrentHashMap<>();
 
-  BonsaiWorldStateUpdater(final BonsaiWorldView world) {
-    this(world, (__, ___) -> {}, (__, ___) -> {});
-  }
+  private boolean isAccumulatorStateChanged;
```

Comment:
Is this thread-safe?

Reply:
It is not modified in a concurrent way so it's good

Reply:
Might be nice to use the `@ThreadSafe` annotation to show it's been considered.